.github/ISSUE_TEMPLATE/onboarding_to_cluster.md

```diff
@@ -8,36 +8,90 @@ assignees: ""
 
 ### Questionnaire
 
-1. **Team name**:
+1. **Target cluster**:
 
-    Your team name can be any name that satisfies [RFC 1123][1]. We will use this name to create an OCP `group` for you, and your team members. If you are only requesting access for yourself, and you do not have a team, you will still need to pick a team name.
+    <!--
+    Please select a cluster from the list at
+    https://github.com/operate-first/apps/tree/master/cluster-scope/overlays/moc#available-clusters
+    -->
 
-2. **Desired namespaces**:
+    TARGET CLUSTER:
 
-    Please provide the names of the `namespaces` you would like created. Names should satisfy [RFC 1123][1]. Please prefix the names with a suitable prefix for your team. We recommend using your team name (as chosen above) if it's already short, or an abbreviation.
+2. **Team name**:
 
-3. **Users needing access**:
+    <!--
+    Your team name should be a short identifier consisting of lower
+    case letters, numbers, and dashes (technically, anything thing
+    satisifies [RFC 1123][1]). For example, `widget-research` is a
+    valid team name, but `Widget Research` is not, because it contains
+    whitespace.
 
-    Please list all users' OCP `username` that require access to the cluster in question.
+    We will use this name to create an OpenShift `group` for you and
+    your team members.
 
-    Note that if you are looking for access to the MOC `zero` cluster, you will first need to log in to the [console][5] using SSO (any Gmail account will work). Doing so will create your user. Your `username` will be the e-mail address you use. Please provide us this value. If this is sensitive, then please reach out to an Operate-First team member and directly message them your username. All this information is encrypted and stored in git.
+    You need to pick a "team name" whether or not you are requesting
+    access for a group of people or an individual.
```

Comment:
Can we add a one liner to discourage people from picking individual names as team names? 
context: https://github.com/operate-first/support/issues/149